### DISCUSSION TITLE
server: enable replacing existing ASCiinema movies by new, improved versions

### DISCUSSION THREAD

**Discussion by pdfkungfoo** [asciinema/discussions#27] [Ideas]
<sub>(Sorry, I don't know how to "re-open" a closed issue, so here is a new one. For previous issue asciinema/asciinema#119 your comment upon closing it was: *"I'm closing old issues. Please reopen if you still think this is a valuable feature worth maintaining."* And I ***DO*** think this.)</sub>

----

Please make it possible for the ASCiinema website to replace one version of a recording by a new one which has error corrections and other improvements **without assigning a new URL** to it. 

This way one could pass the URI of a cool asciicast around without fearing that it may become outdated.

A current workaround to update/improve a recording would be to delete the old version and upload a new one -- but this would also assign a new URI.

**Comment by sebastian-philipp** [comment]
I've got some feedback that my recording was too fast. I'd like to re-upload the recording with an improved version
